docs/ECLLanguageReference/ECLR_mods/BltInFunc-OUTPUT.xml

```diff
@@ -435,7 +435,8 @@
 
               <entry>Optional. The number of days from last file read after
               which the file may be automatically deleted. If omitted, the
-              default is seven (7).</entry>
+              default is seven (7). This overrides the ExpiryDefault in Sasha
+              (The default is 14).</entry>
```

Comment:
> If omitted, the default is seven (7).

That statement is not true and should be removed, if omitted a standard output file does not expire.